scss/response/_object.scss

```
@@ -30,6 +30,21 @@
   color: $light-green;
 }
 
+.street {
+  color: $dark-gray;
+  font-weight: bold;
+}
+
+.next-street {
+  color: $dark-gray;
+  font-weight: bold;
+}
+
+.length {
+  color: $dark-gray;
+  font-weight: bold;
+}
+
```

If you fancy some selector list :)

```suggestion
.street, .next-street, .length {
  color: $dark-gray;
  font-weight: bold;
}
```